helm_deploy/values-dev.yaml

```diff
@@ -23,6 +23,10 @@ generic-service:
     DPS_URL: https://digital-dev.prison.service.justice.gov.uk/
 
   allowlist:
+    ark-nps-hmcts-ttp2: 194.33.192.0/25
```

Comment:
I'd say to add ark-nps-hmcts-ttp1 too, as it is the same group, even it is a different ip address range.

Reply:
added.